### PR TITLE
Flexible severities

### LINKED ISSUE #82: Severity levels may not always exist
Some scenarios may only include `high` or `low` severity levels. Data input cannot guarantee they will all exist so we need to build in catches throughout the code base. 

Also, gray out the toggle when they don't exist.

### PR DESCRIPTION
**Pull request by lxu213**
This PR will refactor the code to allow for some scenarios to have less than the 3 standard severity levels. It should also be prepared for scenarios with different severity levels. 

Fixes #82

### CODE REVIEW

**Comment by lxu213** [comment]
@veev this PR is a work in progress, I started to implement some of the things we need to allow for flexible severity levels, but I got stuck in `Chart`. It looks like `quantileObj` in line 96 of `Chart` is created on a severity level and scenario is a level below, which means structurally it can't take into account severity levels that are different.

I tried flipping the severity and scenario keys of quantileObj but ran into issues in line 141 when they are used to build the Chart. 

Would you be able to take over from here? (There are probably more bugs/issues for this body of work even beyond this, so I can take back over after we cross this `Chart` bridge)

**Comment by lxu213** [comment]
FYI - for testing, you can add this line to  line 70 of `MainContainer.js` so that it's missing a severity level:
`delete dataset['Scenario_A']['med'];`

Although, I haven't even really been using it for the `Chart` issue.

**Comment by veev** [comment]
Will look into the quantileObj issue in Chart @lxu213! After I get a closer look at the issue I'll let you know what could be happening on the draw side.

**Comment by veev** [comment]
I believe this issue has been addressed, at least in the draw side in Chart. Take a look and let me know what you think!

**Comment by lxu213** [comment]
I think there's still an issue - if I remove a scenario from the dropdown in `Chart`, all of the severity levels collapse into one bar. Are you seeing this as well?

Screen Shot 2020-07-11 at 2 41 37 PM

**Comment by veev** [comment]
Ah yes, forgot I needed to do some things for the transition update. Thanks for testing!

**Comment by veev** [comment]
Ok, try testing now, it's working for me!

**Comment by lxu213** [comment]
Great - the `Chart `portions looks ready! I'll add the `Graph` flexibility now (graying out any severity levels that don't exist in the scenario).

**Comment by lxu213** [comment]
@veev it's kind of hard to test, but I think we're ready for flexible severity levels. Can you review?

File: src/components/MainContainer.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import { Layout } from 'antd';
-import { margin, dimMultipliers } from '../utils/constants';
```
Comment (veev) [inline src/components/MainContainer.js:3]: @lxu213 Any reason why you walked back putting the aspect ratio dimensions into the constacts file?

Comment (lxu213) [reply src/components/MainContainer.js]: I have no idea how that happened... which is disturbing, but alas, I've restored this with the latest commits.

**Comment by lxu213** [comment]
The TS bug should be resolved now - let me know if you still have issues compiling.

**Comment by veev** [comment]
@lxu213 this looks good to me. the one thing I'm noticing is that transitioning between the severities looks different on the graph when 2 scenarios are selected. the left side transitions with an opacity fade and the right side doesn't. trying to check on master to see if this is happening on master at the moment, but overwrote the multiple scenarios files. other than that this looks good to me.

**Comment by veev** [comment]
this issue seems to be present on master so will go ahead and approve this PR and hopefully address the minor inconsistency with another issue.

**Pull request merged**